### PR TITLE
Fix performance regression caused by the Bottom App Bar demo

### LINKED ISSUE #16377: "Add a bottom app bar to the floating action button motion demo" regressed missed_frame_build_budget_count
"Add a bottom app bar to the floating action button motion demo" (https://github.com/flutter/flutter/pull/16196) regressed `missed_frame_build_budget_count`.

Please investigate the cause and resolve this issue. Thanks!

(Splitting this off from https://github.com/flutter/flutter/issues/16234)

### PR DESCRIPTION
**Pull request by DaveShuckerow**
Fixes #16377.

Removes the AnimatedCrossFade that was slowing the load time of this demo down.  From testing locally on a Nexus 6p, this should improve the rendering speed by ~30%

### CODE REVIEW

**Review by gspencergoog — APPROVED** [review]

**Comment by gspencergoog** [comment]
32384589-a60f0e74-c078-11e7-9bc1-e5b5287aea9d

File: examples/flutter_gallery/lib/demo/material/bottom_app_bar_demo.dart [hunk 1]
```diff
@@ -271,35 +271,24 @@ class _DemoBottomAppBar extends StatelessWidget {
...
-              firstChild: buildBabContents(context, _BabMode.END_FAB),
-              firstCurve: showsFirst ? fadeOutCurve  : fadeInCurve,
-              secondChild: buildBabContents(context, _BabMode.CENTER_FAB),
-              secondCurve: showsFirst ? fadeInCurve  : fadeOutCurve,
-              crossFadeState: showsFirst ? CrossFadeState.showFirst : CrossFadeState.showSecond,
-            ),
-          ),
-        ],
-      ),
+      // TODO: Use an AnimatedCrossFade to build contents for centered FAB performantly.
```
Comment (Hixie) [inline examples/flutter_gallery/lib/demo/material/bottom_app_bar_demo.dart]: syntax of TODOs is `// TODO(username): description, github_issue_url`

**Comment by Hixie** [comment]
Why not just fix AnimatedCrossFade?

**Comment by Hixie** [comment]
FWIW, this does not appear to have helped the regression.

**Pull request merged**